WaykichainWallet.go

```diff
@@ -1,98 +1,96 @@
 package wiccwallet
 
 import (
-	"wiccwallet/commons"
 	"encoding/hex"
+
+	"wicc-wallet-utils-go/commons"
 )
 
 const WAYKI_TESTNET commons.Network = 1
 const WAYKI_MAINTNET commons.Network = 2
 
-//创建助记词
-func CreateMnemonics() (string){
-   mn:= commons.NewMnemonicWithLanguage(commons.ENGLISH)
-   words,err:=mn.GenerateMnemonic()
-   if err!=nil{
-   	return ""
-   }
-   return words
+//Generate Mnemonics string, saprated by space, language is EN(english)
+func GenerateMnemonics() string {
+	mn := NewMnemonicWithLanguage(ENGLISH)
+	words, err := mn.GenerateMnemonic()
+	if err != nil {
+		return ""
+	}
+	return words
 }
 
 //助记词转换地址
-func Mnemonic2Address(words string,netType commons.Network)(string){
-	address := commons.GenerateAddress(words,netType)
+func Mnemonic2Address(words string, netType commons.Network) string {
```

Comment:
naming better be changed as : GetAddressFromMnemonic

Reply:
Maybe all of the same style functions in this file should be rename like "GetXxxFromYyy"?